### DISCUSSION TITLE
how i can scollect uwsgi + django trace data with sw-python

### DISCUSSION THREAD

**Discussion by Feng-Wuhui** [apache/skywalking#9722] [Q&A]
hi， every one, i want to know that ,the python agent sw-python can collect the uwsgi+django trace data in cli mode Now ? It looks like the sw-python can not support the uwsgi+django to collect trace data? please help me ,thank you

**Comment by wu-sheng** [comment]
If you keep posting repeated things, I have to block you and lock all your post.
